portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -13,14 +13,14 @@ public class FormHandlerServlet extends HttpServlet {
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     // Get the value entered in the form.
-    String textValue = request.getParameter("text-input");
+    String textValueName = request.getParameter("text-input-name");
+    String textValueEmail = request.getParameter("text-input-email");
 
     // Print the value so you can see it in the server logs.
-    System.out.println("You submitted: " + textValue);
+    System.out.println(textValueName + " submitted contact email: " + textValueEmail);
 
     // Write the value to the response so the user can see it.
-    response.getWriter().println("You submitted: " + textValue);
-
+    response.getWriter().println(textValueName + " submitted contact email: " + textValueEmail);
     response.sendRedirect("https://zhe-sps-summer22.appspot.com/");
   }
```

Comment:
Where is the response displayed?

In later versions, if you decide to persis the info, you should check if check if the textValueName is in email format. But for now this is ok.